isic_challenge_scoring/task3.py

```diff
@@ -56,6 +56,7 @@ def compute_metrics(truth_file_stream, prediction_file_stream) -> Dict[str, Dict
             'ap': metrics.average_precision(
                 truth_category_probabilities, prediction_category_probabilities
             ),
+            # 'roc': metrics.roc(truth_category_probabilities, prediction_category_probabilities),
```

Comment:
Why isn't this being used? Why do we need to set it in another loop below?

Reply:
If I put it in that loop, the macro_average tries to calculate the average of the roc curve and throws an error. I couldn't figure out how to make the macro_average skip roc so I thought to just add another loop at the end to add the roc metric in.